cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,591 @@
+
+            - VSA (Vessel Sharing Agreement)
+            - SCA (Slot Charter Agreement)
+      required:
+        - code
+        - codeListProvider
+        - agreementTypeCode
+    VesselBooking:
+      title: VesselBooking
+      x-stoplight:
+        id: 262a2bff1896c
+      type: object
+      properties:
+        vesselOperatorSMDGLinerCode:
+          type: string
+          description: The carrier who is in charge of the vessel operation based on the SMDG code
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 10
+          example: HLC
+        vesselIMONumber:
+          type: string
+          pattern: '^\d{7}$'
+          maxLength: 7
+          example: '9321483'
+          description: 'The unique reference for a registered Vessel. The reference is the International Maritime Organisation (IMO) number, also sometimes known as the Lloyd''s register code, which does not change during the lifetime of the vessel'
+        vesselName:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 35
+          example: King of the Seas
+          description: The name of the Vessel given by the Vessel Operator and registered with IMO.
+        forecasts:
+          type: array
+          items:
+            $ref: '#/components/schemas/Forecast'
+      required:
+        - vesselOperatorSMDGLinerCode
+        - vesselIMONumber
+        - forecasts
+    Forecast:
+      title: Forecast
+      x-stoplight:
+        id: 299e1a5e0f756
+      type: object
+      properties:
+        portVisitReference:
+          type: string
+          description: The unique reference that can be used to link different transportCallReferences to the same port visit. The reference is provided by the port to uniquely identify a port call.
+          pattern: ^\S+(\s+\S+)*$
+          example: NLRTM1234589
+          maxLength: 50
+        transportCallReference:
+          type: string
+          description: |-
+            A carrier definied reference to a TransportCall.
+
+            In the case the Means of Transport is a Vessel and the facility is a Port/Terminal - this reference should be considered a Terminal Call Reference
+          example: da97d1e3-e546-4647-891b-3f57e04b886b
+        carrierImportVoyageNumber:
+          type: string
+          example: 2103N
+          description: The identifier of an import voyage. The carrier-specific identifier of the import Voyage.
+          maxLength: 50
+        carrierExportVoyageNumber:
+          type: string
+          description: The identifier of an export voyage. The carrier-specific identifier of the export Voyage.
+          example: 2103S
+          maxLength: 50
+          pattern: ^\S+(\s+\S+)*$
+        universalImportVoyageReference:
+          type: string
+          description: 'A global unique voyage reference for the import Voyage, as per DCSA standard, agreed by VSA partners for the voyage.'
+          pattern: '^\d{2}[0-9A-Z]{2}[NEWS]$'
```

Comment:
Do you need to add the "R" for Roundtrip?

Reply:
Kept is as decided in the past, there is no decision on the  "R" yet.